ci/test_wheel_cuspatial.sh

```diff
@@ -5,15 +5,19 @@ set -eou pipefail
 
 mkdir -p ./dist
 RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
-RAPIDS_PY_WHEEL_NAME="cuspatial_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./dist
 
-# Install additional dependencies
+# install build dependencies for fiona
 apt update
 DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends libgdal-dev
```

Comment:
Is the question why we install fiona with `--no-binary`? If so, I do think arm is one of the reasons, but it may not be the only one. @trxcllnt might remember more.

Reply:
I don't know the original reason, not having ARM packages seems like a likely candidate.

This PR doesn't change anything about it (it's just showing up in the diff because I'm moving other install-related code around), so I'm going to treat this discussion as non-blocking.